pkg/metrics/metric.go

```diff
@@ -44,15 +44,6 @@ func (a APIMetricType) MarshalText() ([]byte, error) {
 
 // UnmarshalText is a custom unmarshaller for APIMetricType (used for testing)
 func (a APIMetricType) UnmarshalText(buf []byte) error {
-	tmp := string(buf)
```

Comment:
Not sure what this function is supposed to do, but removing it crashes some tests

Reply:
cc @gmmeyer can you provide more context? 🙏